.make/python.mk

```diff
@@ -7,18 +7,15 @@ PYTHON_PATH=""
 .help-python:
 	@echo ""
 	@echo "$(SMUL)$(BOLD)$(GREEN)Python$(SGR0)"
-	@echo "  $(BOLD)python-test$(SGR0) -- Run 'yapf -r -i -vv --style google --exclude '**/.cargo/registry' .'"
-	@echo "                 to validate python files against Google style guide."
-	@echo "                 Run 'flake8 --exclude '**/.cargo/registry' .' to validate python files against flake8 style guide."
-	@echo "  $(BOLD)python-tidy$(SGR0) -- Run 'black --extend-exclude .cargo ' to fix python style formats if needed."
+	@echo "  $(BOLD)python-test$(SGR0) -- Run 'pycodestyle --exclude='**/.cargo' --exclude='**/target' .'"
+	@echo "                 to validate python files against pep8 style guide."
+	@echo "  $(BOLD)python-tidy$(SGR0) -- Run 'autopep8 --in-place --recursive --exclude='**/.cargo' --exclude='**/target' .' to fix python style formats if needed."
 
-# Python / yapf, flake8 targets
+# Python / pep8 targets
 python-test: docker-pull
-	@echo "$(CYAN)Formatting and checking Python files with Google style...$(SGR0)"
-	@$(call docker_run,yapf -r -i -vv --style google --exclude "$(PYTHON_PATH).cargo/registry" .)
-	@echo "$(CYAN)Formatting and checking Python files with flake8 style...$(SGR0)"
-	@$(call docker_run,flake8 --exclude "$(PYTHON_PATH).cargo/registry" .)
+	@echo "$(CYAN)Formatting and checking Python files with pep8 style...$(SGR0)"
+	@$(call docker_run,pycodestyle --exclude="$(PYTHON_PATH).cargo" --exclude="$(PYTHON_PATH)target" .)
 
 python-tidy: docker-pull
 	@echo "$(CYAN)Running python file formatting fixes...$(SGR0)"
-	@$(call docker_run,black --extend-exclude $(PYTHON_PATH).cargo .)
+	@$(call docker_run, autopep8 --in-place --recursive --exclude="$(PYTHON_PATH).cargo" --exclude="$(PYTHON_PATH)target" .)
```

Comment:
```mk
	@echo "  $(BOLD)python-test$(SGR0) -- Run 'pycodestyle --exclude='**/.cargo' .'"
	@echo "                 to validate python files against pep8 style guide."
	@echo "  $(BOLD)python-tidy$(SGR0) -- Run 'autopep8 --in-place --recursive --exclude='**/.cargo' .' to fix python style formats if needed."

# Python / pep8 targets
python-test: docker-pull
	@echo "$(CYAN)Formatting and checking Python files with pep8 style...$(SGR0)"
	@$(call docker_run,pycodestyle --exclude="$(PYTHON_PATH).cargo" .)

python-tidy: docker-pull
	@echo "$(CYAN)Running python file formatting fixes...$(SGR0)"
	@$(call docker_run, autopep8 --in-place --recursive --exclude="$(PYTHON_PATH).cargo" --exclude="$(PYTHON_PATH)target" .)
```

Reply:
This was updated here: https://github.com/Arrow-air/tf-github/pull/109

Reply:
Hmm, wonder why this didn't get into the branch. I see it's there on `develop` but I might have created the branch before that. I'll fix it

Reply:
hmm I'm still seeing the same thing. For python-test, there should only be one exclude flag.

Reply:
The content is the same as https://github.com/Arrow-air/tf-github/blob/main/src/templates/all/.make/python.mk
There are 2 exclude tags there as well (for different matches)

Reply:
Oh! nvm, I missed you were linking to an open PR, I missed that one!

Reply:
Should be really fixed now ;-)